readme.md

```diff
@@ -41,7 +41,7 @@ var theBody = document.querySelectorAll("body");
 theBody.style.backgroundColor = "red";
 ```
 
-> Your answer...
+> Erm... it needs to be .querySelector? Not .querySelectorAll
```

Comment:
:+1: querySelectorAll returns an array, so you'd have to fetch the first element from the array, or as you said, use `querySelector`.